docs/whats-new/staging-release-notes.md

```diff
@@ -1,895 +1,295 @@
 # Staging release notes
 
-## Staging release 7 Jun 2023
+## July 2023
 
-Frontend version: 1.0.0-20230529.1337 | Backend version: 1.64.2-230612.0513
-
-**Changes** 
-
-<details>
-<summary> MFA fraud alert is now enabled.
-</summary>
+**5 July 2023**
 
-The MFA fraud alert feature is now enabled to enhance security on TechPass Portal. This allows users to report potential fraud incidents related to Multi-Factor Authentication (MFA) challenges. Refer to  [MFA fraud alert](https://stg.docs.developer.tech.gov.sg/docs/techpass-user-guide/mfa-fraud-alert) for more information.
-</details>
+Frontend version: 11.0.0-20230630.1455 | Backend version: 1.69.0-20230630.1619
 
-## Staging release 24 May 2023
-
-Frontend version: 1.0.0-20230520.0010 | Backend version: 1.62.1-230522.1016
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** | *Get* and *Delete* application is now available on **Automation API**. | This new feature on the **Automation API** enables tenants to automate the deletion of applications and retrieve detailed information about a specific application within their environment. |
+| **Enhancement** | Enhanced self-sign up page with eligibility checks. | The enhanced self-sign up page with eligibility checks is now available to all users on the TechPass portal. This feature ensures a smoother onboarding process and improved user guidance based on email domain eligibility. |
+| **Enhancement** | Webhook domain whitelist update in staging environment. | The staging environment now accepts webhooks originating from *wogaa.cloud*, *ngrok-free.app*, *ngrok-free.dev*, *ngrok.app* and *ngrok.dev* domains. Webhooks sent from these domains will be processed and handled within the staging environment. |
```

Comment:
a reminder that this point is to be removed from prod rn

Reply:
got it